### PR TITLE
Tile entity protocol + Sign and Banner

### PR DESCRIPTION
**Pull request by Cybermaxke**
Adds tile entity protocol and implements this for the available tile entities, additionally implementing the Sign.

### CODE REVIEW

File: src/main/java/org/lanternpowered/server/game/registry/type/item/ItemRegistryModule.java [hunk 1]
```diff
@@ -646,6 +649,10 @@ public void registerDefaults() {
         register(323, builder()
                 .translation("item.sign.name")
                 .maxStackQuantity(16)
+                .behaviors(pipeline -> pipeline
+                        .add(WallOrStandingPlacementBehavior.ofTypes(
+                                () -> BlockTypes.WALL_SIGN,
+                                () -> BlockTypes.STANDING_SIGN)))
```
Comment (Meronat) [inline src/main/java/org/lanternpowered/server/game/registry/type/item/ItemRegistryModule.java]: You need some sort of behavior to send a `MessagePlayOutOpenSign` here.

Comment (Cybermaxke) [reply src/main/java/org/lanternpowered/server/game/registry/type/item/ItemRegistryModule.java]: Yeah, right now it's right click to open a sign.

Comment (Meronat) [reply src/main/java/org/lanternpowered/server/game/registry/type/item/ItemRegistryModule.java]: I noticed that, it's a nice touch.

**Comment by Cybermaxke** [comment]
Merged in https://github.com/LanternPowered/LanternServer/commit/772f1b0fe8badae24adad446e65150c65143772c

**Pull request closed without merging**